backend/wmg/data/load_cube.py

```diff
@@ -13,37 +13,186 @@
 
 stack_name = os.environ.get("REMOTE_DEV_PREFIX")
 wmg_bucket_name = os.environ.get("WMG_BUCKET")
-artifact_bucket_name = os.environ.get("ARTIFACT_BUCKET")
```

Comment:
was this previously just not being used at all? might be nice to also remove it from wherever we set the environment variables. not a big deal though